inspectors/models/score/barssettingsmodel.cpp

```diff
@@ -0,0 +1,74 @@
+#include "barssettingsmodel.h"
+
+BarsSettingsModel::BarsSettingsModel(QObject* parent, IElementRepositoryService* repository) :
```

Comment:
Here and all others too, `BarSettingsModel`.

Reply:
or `MeasureSettingsModel`